### ISSUE TITLE
Unsure of behaviour when package is installed directly from source control

### ISSUE THREAD

**Issue by steven-murray** [RadioAstronomySoftwareGroup/pygitversion#1]
In the README it has the line:

```
Package installed directly from hosted source control via ``pip install git+git:...``: unsure?
```

This behavior should be verified. @bhazelton do you have any idea how it should work in this case?

As a bonus, any ideas for other cases that need to be covered?

**Comment by bhazelton** [comment]
I agree it should be verified. I'm not sure I agree that the behavior you list with `pip install -e .` is what happens (and I'm pretty sure it's not what we want to happen). I think when you install that way you want to get the git hash of the currently running code NOT the hash that existed when you ran `pip install -e .`. I believe that is what happens because the code first tries to get the hash from git before it reads the `GIT_INFO` file.

**Comment by steven-murray** [comment]
This has been fixed, and now works as intended, though I cannot prove it with tests...
